ingest/ingest_pipeline.py

```diff
@@ -26,8 +26,8 @@
 # Ingest dense file
 python ingest_pipeline.py --study-id 5d276a50421aa9117c982845 --study-file-id 5dd5ae25421aa910a723a337 ingest_expression --taxon-name 'Homo sapiens' --taxon-common-name human --ncbi-taxid 9606 --matrix-file ../tests/data/dense_matrix_19_genes_1000_cells.txt --matrix-file-type dense
 
-# Ingest loom file
-python ingest_pipeline.py  --study-id 5d276a50421aa9117c982845 --study-file-id 5dd5ae25421aa910a723a337 ingest_expression --matrix-file ../tests/data/test_loom.loom  --matrix-file-type loom --taxon-name 'Homo Sapiens' --taxon-common-name human
+# Ingest h5ad file
+python ingest_pipeline.py  --study-id 5d276a50421aa9117c982845 --study-file-id 5dd5ae25421aa910a723a337 ingest_h5ad --h5ad-file ../tests/data/test.h5ad
```

Comment:
Out-of-scope here, but maybe worthwhile as part of the larger "simplify Ingest Pipeline" initiative this quarter:

The use of both different subparsers (e.g. `ingest_expression`, `ingest_h5ad`) _and_ different argument names for "top level file" (e.g. `--matrix-file`, `--h5ad-file`) in each subparser seems unnecessarily complex.  Would be simpler to have a consistently-named argument for top-level file (e.g. `--file`), and encapsulate the business rules around subparser-specific file validation into the respective subparser argument handling?

Reply:
I agree that simplifying the number of argument names could simplify things but for some ingest use cases, there is more than one "top-level file". Subsampling requires both cluster and metadata files. DE expects cluster, metadata and matrix files. 

Also the current business rules around handling files is (mostly) consolidated in the IngestFile class. Simplification as suggested above would entail implementing a different overall design. I'm not adverse to it, but it's out of scope for this PR for sure.